src/test/java/com/amplitude/AmplitudeTest.java

```diff
@@ -55,6 +57,14 @@ public void testLogEventSuccess(boolean useBatch)
               latch.countDown();
               return response;
             });
+    List<AmplitudeEventCallback> eventCallbacks = new ArrayList<>();
```

Comment:
It doesn't look like this is added via setEventCallback(). Also I don't we need the array anymore.

Reply:
Good catch! That was accidentally left inside tests. Let me remove and clean up this test.